widgetssdk/src/main/res/values/dimens.xml

```diff
@@ -9,6 +9,7 @@
     <dimen name="glia_large_x_large">24dp</dimen>
     <dimen name="glia_x_large">32dp</dimen>
     <dimen name="glia_xx_large">64dp</dimen>
+    <dimen name="glia_touch_area_min_size">44dp</dimen>
```

Comment:
Based on [what I see](https://support.google.com/accessibility/android/answer/7101858?hl=en), it should be 48dp, am I right?

> Consider making touch targets at least 48x48dp, separated by 8dp of space or more, to ensure balanced information density and usability. A touch target of 48x48dp results in a physical size of about 9mm, regardless of screen size.